Packs/CyberArkAIMv2/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py

```diff
@@ -0,0 +1,164 @@
+    """Lists all credentials available.
+    :param client: the client object with the given params
+    :return: the credentials info without the explicit password
+    """
+    res = client.list_credentials()
+    # the password value in the json appears under the key "Content"
+    if "Content" in res:
+        del res["Content"]
+    # notice that the raw_response doesn't contain the password either
+    results = CommandResults(
+        outputs=res,
+        raw_response=res,
+        outputs_prefix='CyberArkAIM',
+        outputs_key_field='Name',
+    )
+    return results
+
+
+def fetch_credentials(client):
+    """Fetches the available credentials.
+    :param client: the client object with the given params
+    :return: a credentials object
+    """
+    res = client.list_credentials()
+
+    credentials = {
+        "user": res.get("UserName"),
+        "password": res.get("Content"),
+        "name": res.get("Name"),
+    }
+    demisto.credentials([credentials])
+
+
+def test_module(client: Client) -> str:
+    """Performing a request to the AIM server with the given params
+    :param client: the client object with the given params
+    :return: ok if the request succeeded
+    """
+    client.list_credentials()
+    return "ok"
+
+
+def main():
+
+    params = demisto.params()
+
+    url = params.get('url')
+    use_ssl = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+
+    app_id = params.get('app_id') or ""
+    folder = params.get('folder')
+    safe = params.get('safe')
+    credential_object = params.get('credential_names') or ""
+
+    cert_text = params.get('cert_text') or ""
+    key_text = params.get('key_text') or ""
+
+    username = ""
+    password = ""
+    if params.get('credentials'):
+        # credentials are not mandatory in this integration
+        username = params.get('credentials').get('identifier')
+        password = params.get('credentials').get('password')
+
+    try:
+        client = Client(server_url=url, use_ssl=use_ssl, proxy=proxy, app_id=app_id, folder=folder, safe=safe,
+                        credential_object=credential_object, username=username, password=password,
+                        cert_text=cert_text, key_text=key_text)
+
+        command = demisto.command()
+        demisto.debug(f'Command being called in CyberArk AIM is: {command}')
+
+        commands = {
+            'test-module': test_module,
+            'cyberark-aim-list-credentials': list_credentials_command,
+            'fetch-credentials': fetch_credentials
+        }
+        if command in commands:
+            return_results(commands[command](client))  # type: ignore[operator]
+        else:
+            raise NotImplementedError(f'{command} is not an existing CyberArk PAS command')
+    except Exception as err:
+        return_error(f'Unexpected error: {str(err)}', error=traceback.format_exc())
+    finally:
+        try:
+            if client.crt:
+                client.cf.close()
```

Comment:
Close is not enough. You still need to delete the file. Also, check if cf is not None.